tests/test_converters.py

```diff
@@ -292,6 +293,70 @@ def test_converter(self):
                      ('batch', 'channel', 'height', 'width'))
 
 
+class TestCelebA(object):
+    def setUp(self):
+        numpy.random.seed(21 + 1 + 2016)
+        self.tempdir = tempfile.mkdtemp()
+        cwd = os.getcwd()
```

Comment:
#302 squashes and rebases #265. If I can manage to get it merged soon I'll use the context manager for this PR.